src/components/dialogs/create-case-dialog/create-case-dialog-utils.ts

```diff
@@ -0,0 +1,14 @@
+import yup from '../../utils/yup-config';
+import { CASE_FILE, CASE_NAME, DESCRIPTION } from '../../utils/field-constants';
+
+export const getCreateCaseDialogFormValidationDefaultValues = () => ({
+    [CASE_NAME]: '',
+    [DESCRIPTION]: '',
+    [CASE_FILE]: null,
+});
+
+export const createCaseDialogFormValidationSchema = yup.object().shape({
+    [CASE_NAME]: yup.string().required(),
+    [DESCRIPTION]: yup.string(),
+    [CASE_FILE]: yup.mixed(),
```

Comment:
```suggestion
    [CASE_FILE]: yup.mixed().nullable,
```
since CASE_FILE can be null